src/test/groovy/com/vanniktech/android/junit/jacoco/ProjectHelper.groovy

```diff
@@ -4,11 +4,11 @@ import com.android.build.gradle.AppExtension
 import com.android.build.gradle.LibraryExtension
 import com.android.build.gradle.TestExtension
 import com.android.build.gradle.api.BaseVariant
+import com.android.build.gradle.internal.coverage.JacocoOptions
 import com.android.builder.model.BuildType
 import groovy.mock.interceptor.MockFor
 import org.gradle.api.Project
 import org.gradle.testfixtures.ProjectBuilder
-
```

Comment:
nit: can we keep this new line?

Reply:
I think the provided editor config is messing with this.